regression/pageObjects/productCategories.js

```diff
@@ -0,0 +1,197 @@
+class productCategories {
+  
+    async clickFashion(){
```

Comment:
`    async clickFashion(id){
         let fashion = element(by.id("#{id}"));
         var EC = protractor.ExpectedConditions;
         browser.wait(EC.elementToBeClickable(fashion), 5000);
         await fashion.click();
     };`

Could we write it like this?